charts/testkube-operator/values.yaml

```diff
@@ -29,6 +29,47 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+webhook:
+  enabled: true
+  name: testkube-operator-webhook-admission
+  labels: {}
+  annotations: {}
+
+  patch:
+    enabled: true
+    image:
+      repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
```

Comment:
should it be just hardcoded?

Reply:
I don't have a strong opinion.
I followed the pattern how Prometheus Operator and NGINX Controller use the same tool for certgen and admission patch.

Reply:
ok